src/corehost/cli/hostfxr.h

```diff
@@ -26,54 +30,55 @@ enum class hostfxr_delegate_type
     load_assembly_and_get_function_pointer
```

Comment:
Thoughts on giving this a prefix so that it is less likely to collide in C?
`hdt_`? (`hostfxr_delegate_type` seems a bit too long)

Reply:
Yeah - we'll need one... "hdt" is weird... but others are probably as weird anyway.
The only other idea I have is go long - so `host_delegate_load_assembly_and_get_function_pointer` - but that is really long...
I'm fine with `hdt` I guess.